### ISSUE TITLE
-threaded assumes Intel MPI

### ISSUE THREAD

**Issue by boegel** [gjbex/worker#46]
Using `wsub -threaded` results in using `mpirun -ppn X`, which is specific to Intel MPI.

With OpenMPI, you get:

```
mpirun: Error: unknown option "-ppn"
```

For OpenMPI, the equivalent would be `mpirun --map-by ppr:4:node`.

**Comment by gjbex** [comment]
This is not the only dependency on Intel MPI, do you have a specific reason to avoid it in favor of Open MPI?

**Comment by boegel** [comment]
Not necessarily, I just ran into it with our worker installation on top of `foss`, I'll proceed with installing it with the `intel` toolchain instead to dance around this.

If `worker` assumes Intel MPI, maybe it should somehow check that `mpirun` is indeed provided by Intel MPI, to avoid surprises?

**Comment by gjbex** [comment]
Hm, the configure.ac is rather explicit about it:
AC_PROG_CC([mpiicc])

Also keep in mind that worker-ng is just around the corner.
